extension/packages/nextjs/layout.tsx.args.mjs

```diff
@@ -0,0 +1 @@
+export const globalCssImports = 'import "@coinbase/onchainkit/styles.css";';
```

Comment:
ohh I think this should be `export const imports=` because we renamed the variable in https://github.com/scaffold-eth/create-eth/pull/74/files#diff-d447c6d93270febc35e0430dd2b7dd92fcd104ff952eef1c5c1f753f5a31eb85

Reply:
You're right! Also moved file since it was in wrong folder [6eff85b](https://github.com/scaffold-eth/create-eth-extensions/pull/11/commits/6eff85bf598a11e5356ac0553c5979447132b790)